### PR TITLE
search past searches and loading

### PR DESCRIPTION
**Pull request by kevingreer**
Shows past searches when you first tap the search bar, and has a loading animation when fetching data

### CODE REVIEW

File: Podcast/PastSearchTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+class PastSearchTableViewCell: UITableViewCell {
+    
+    var label: UILabel!
+    var iconImageView: UIImageView!
+    
+    let XEdgePadding: CGFloat = 18
```
Comment (dennisfedorko) [inline Podcast/PastSearchTableViewCell.swift]: xEdgePadding

File: Podcast/PastSearchTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    let YEdgePadding: CGFloat = 18
```
Comment (dennisfedorko) [inline Podcast/PastSearchTableViewCell.swift]: yEdgePadding

File: Podcast/TabbedPageViewController.swift [hunk 3]
```diff
@@ -282,4 +326,38 @@ class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource
...
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "PastSearchCell") as? PastSearchTableViewCell else { return UITableViewCell() }
+        guard let pastSearches = UserDefaults.standard.array(forKey: "PastSearches") as? [String] else {
+            cell.configureNoPastSearches()
+            return cell
+        }
+        cell.label.text = pastSearches[indexPath.row]
+        return cell
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
```
Comment (dennisfedorko) [inline Podcast/TabbedPageViewController.swift]: you don't need this, default is 1 section

File: Podcast/TabbedPageViewController.swift [hunk 4]
```diff
@@ -282,4 +326,38 @@ class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource
...
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if let pastSearches = UserDefaults.standard.array(forKey: "PastSearches") as? [String] {
```
Comment (dennisfedorko) [inline Podcast/TabbedPageViewController.swift]: return UserDefaults.standard.array(forKey: "PastSearches") ? pastSearches.count : 1

**Comment by dennisfedorko** [comment]
looks good, just address those few comments and merge

File: Podcast/TabbedPageViewController.swift (hunk 4 above)
Comment (kevingreer) [reply Podcast/TabbedPageViewController.swift]: pastSearches is only unwrapped after the if let though

**Pull request merged**